bundles/org.openhab.core.test.magic/src/main/java/org/openhab/core/magic/binding/internal/automation/modules/MagicSingleActionService.java

```diff
@@ -33,7 +33,7 @@
  *
  * @author Stefan Triller - Initial contribution
  */
-@Component(configurationPid = "org.openhab.core.automation.action.magicSingleActionService", property = {
+@Component(configurationPid = "org.openhab.automation.action.magicSingleActionService", property = {
```

Comment:
```suggestion
@Component(configurationPid = "org.openhab.magicsingleactionservice", property = {
```

Reply:
I changes this one to: `"org.openhab.magicsingleaction"`